src/builders/util.js

```diff
@@ -0,0 +1,5 @@
+export function findNearestElement(view, selector) {
+  if (!view || view.matches("html")) return null;
+  if (!selector) return null;
+  return document.querySelector(selector) || findNearestElement(view.parentElement, selector);
```

Comment:
Before, this would have been `app.getApplicationElement`. We probably still need a finer-grained view boundary to search within than `document`, however, the reason for `document` at the present moment is because I am intentionally trying to eliminate the need for a global application object and I am intentionally trying to minimize the dependencies between various parts of the system.